server/form-pages/utils/matchingInformationUtils.test.ts

```diff
@@ -500,58 +500,85 @@ describe('matchingInformationUtils', () => {
       ;(placementDurationFromApplication as jest.Mock).mockReturnValueOnce(12)
     })
 
-    describe('when the start date is the same as the release date', () => {
-      it('returns the suggested stay from the application as summary list options, using the release date as the start date', () => {
-        when(retrieveQuestionResponseFromFormArtifact)
-          .calledWith(application, PlacementDate, 'startDateSameAsReleaseDate')
-          .mockReturnValue('yes')
-        when(retrieveOptionalQuestionResponseFromFormArtifact)
-          .calledWith(application, ReleaseDate)
-          .mockReturnValue('2024-03-07T00:00:00Z')
+    describe('if the release date is known', () => {
+      describe('when the start date is the same as the release date', () => {
+        it('returns the suggested stay from the application as summary list options, using the release date as the start date', () => {
+          when(retrieveQuestionResponseFromFormArtifact)
+            .calledWith(application, ReleaseDate, 'knowReleaseDate')
+            .mockReturnValue('yes')
```

Comment:
This could be in a `beforeEach` for this `describe` block

Reply:
great shout, automerge got there before I could unfortunately

Reply:
Yeah it was too minor for me to flag. I can sort that next time I edit the page if I remember